aws/resource_aws_elasticsearch_domain_test.go

```diff
@@ -1065,6 +1101,36 @@ resource "aws_elasticsearch_domain" "test" {
 `, rName, zoneAwarenessEnabled)
 }
 
+func testAccESDomainConfigWarm(rName string, enabled bool, count int) string {
+	return fmt.Sprintf(`
+resource "aws_elasticsearch_domain" "test" {
+  domain_name           = %[1]q
+  elasticsearch_version = "6.8"
+
+  cluster_config {
+    zone_awareness_enabled   = true
+    instance_type            = "r5.large.elasticsearch"
```

Comment:
This and the master type seem overly large for running an acceptance test. Any reason not to use the smallest available cluster to test the ultrawarm stuff?

From the [docs](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/ultrawarm.html#ultrawarm-pp) it looks like you can't have t2 nodes as data nodes and must have some form of master nodes to be able to use ultrawarm so I think the [cheapest](https://aws.amazon.com/elasticsearch-service/pricing/) cluster size here is a single AZ c5.large data node and a t2.micro master node.

Reply:
will test these

Reply:
AWS Console doesn't allow creation of ultrawarm nodes with t2 master nodes, as well.

Reply:
changed, and as @ace03uec said. master node cannot be t2 for ultrawarm